### PR TITLE
Update with ide and framework

### PR DESCRIPTION
**Pull request by ArtifexEt**
- I think we can not forget about Angular, because is still used and more popular than Vue.
- Webstorm IDE is not for free but is worth knowing
- Added git ignore.

### CODE REVIEW

**Comment by helloroman** [comment]
Lol, Szymon i'm so sorry that i didn't respond to that – i had no idea that people will create PRs :D

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,123 @@
+### IntelliJ project files
```
Comment (helloroman) [inline .gitignore]: I don't think that gitignore is needed in this repo – there will be only one file here – README.md

**Review by helloroman — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (ArtifexEt) [reply .gitignore]: Removed :)

**Comment by ArtifexEt** [comment]
> Lol, Szymon i'm so sorry that i didn't respond to that – i had no idea that people will create PRs :D

I taught frontend on bootcamp and found your list. it is quite useful thanks ;)

**Review by helloroman — APPROVED** [review]

**Comment by helloroman** [comment]
All good, i'm merging the PR :) Thanks

**Pull request merged**